src/util/MemorySize/MemorySize.cpp

```diff
@@ -58,12 +59,12 @@ std::string MemorySize::asString() const {
 
 // _____________________________________________________________________________
 MemorySize MemorySize::parse(std::string_view str) {
-  if (auto matcher =
-          ctre::match<"(\\d+(?:\\.\\d+)?) ([kKmMgGtT][bB]?|[bB])">(str)) {
-    auto amountString = matcher.get<1>().to_view();
+  if (auto matcher = ctre::match<
+          "(?<amount>\\d+(?:\\.\\d+)?) ?(?<unit>[kKmMgGtT][bB]?|[bB])">(str)) {
```

Comment:
isn't `\\w*` (if that's the correct syntax for whitespace, I always have to look it up) a little bit more generic here than ` ?` ? One could argue that it's not super important, but if we have rather generic error messages, then we want to make the parser as forgiving as possible, especially in places where it doesn't matter.

Reply:
`\w` does match all kinds of whitespace ` ` just the default space. In most cases this doesn't really make a difference, but both is fine here.